medusa/providers/torrent/html/morethantv.py

```diff
@@ -235,23 +235,27 @@ def _parse_season(self, row, download_url, title):
         details_url = row.find('span').find_next(title='View torrent').get('href')
         torrent_id = parse_qs(download_url).get('id')
         if not all([details_url, torrent_id]):
+            logger.log("Could't parse season pack details page for title: %s", title, logger.DEBUG)
```

Comment:
This style doesn't work.
Our log: `def log(self, msg, level=INFO, *args, **kwargs):`

Standard logging: `def log(self, level, msg, *args, **kwargs):`